cafebabel/templates/articles/translations/create.html

```diff
@@ -16,18 +16,18 @@
     {% endif %}
     <h1 class="edit active">
       <label for=title>Title</label>
-      <input id=title name=title required placeholder="Translation in {{ language }} for article “{{ article.title }}”">
+      <input id=title name=title required placeholder="{{ article.title }}">
```

Comment:
I'm getting back at this 🙈 

To me a placeholder is to give an example of what could be typed, here I think the title of the original article is valuable to stay during edition to be able to translate it while seeing it. Proposition: putting it within the `label` like that `<label for=title>Title (original: {{ article.title }})</label>` maybe even with a link to the article in order to be able to access it quickly given there is no text to translate on that page.

Reply:
hum it seems to me more confusing to have the original title as a `label`. I will leave it like that for now, and wait for feedback from users. But thanks for your proposition 😉